contracts/base/ATM/ATMToken.sol

```diff
@@ -0,0 +1,153 @@
+pragma solidity ^0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20Mintable.sol";
+import "@openzeppelin/contracts/lifecycle/Pausable.sol";
+import "./ATMTokenInterface.sol";
+
+/**
+  *  @title ATM Token for Teller DAO
+  *
+  *  @author develop@teller.finance
+ */
+
+contract ATMToken is ATMTokenInterface, ERC20Detailed, Pausable, ERC20Mintable {
+    /**
+      *  @notice ATMToken implements an ERC20 token with a supply cap and a vesting scheduling
+     */
+
+    /* State Variables */
+    uint256 private _cap;
+
+    /* Structs */
+    struct VestingTokens {
+        address beneficiary;
+        uint256 amount;
+        uint256 deadline;
+    }
+
+    /* Mappings */
+    mapping (address => VestingTokens) private _vestingBalances;
+
+    /* Constructor */
+    constructor (
+        uint256 cap
+    ) public
+      ERC20Detailed("ATMToken", "ATM", 18)
+    {
+        _cap = cap;
+    }
+
+    /* Functions */
+    /**
+     * @notice Returns the cap on the token's total supply
+     * @return The supply capped amount
+     */
+    function cap() public view returns (uint256) {
+        return _cap;
+    }
+
+    /**
+     * @notice Sets a new cap on the token's total supply.
+     * @param newCap The new capped amount of tokens
+     */
+    function setCap(uint256 newCap) public onlyPauser() whenNotPaused() {
+        _cap = newCap;
+        emit NewCap(_cap);
+    }
+
+    /**
+     *  @notice Checks if account has a vesting schedule
+     *  @param account The account being checked
+     *  @return true if successful
+     * 
+     */
+    function isVested(address account) public returns (bool) {
+        require(_vestingBalances[account].amount > 0, "ACCOUNT_DOESNT_HAVE_VESTING");
+        return true;
+     }
+
+    /**
+     * @notice Increase account supply of specified token amount
+     * @param account The account to mint tokens for
+     * @param amount The amount of tokens to mint
+     * @return true if successful
+     */
+    function mint(address account, uint256 amount) public onlyPauser() whenNotPaused() returns (bool) {
```

Comment:
Who will able to pause it? Is it possible to inject the Settings contract (using the SettingsInterface and use settings.isPaused() function instead? 

@Juan-cc @passabilities  Thoughts?

Reply:
It would be
- Create the modifier and use the settings.isPaused() function
- Remove the Pausable reference.

Reply:
Similar to the isPauserRole

Reply:
In the future we will have several ATMs and Settings will let us stop all ATMs or none, but we can't pause only 1 (unless we add each ATM address on settings). 
For now I think having a centralized place (Settings or ATMSettings) to deal with this is fine. 
I'll change ATMGovernance to follow this same pattern as I was using Pausable capability.